src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -13,11 +13,18 @@
     <AssemblyName>System.Text.Encoding.CodePages</AssemblyName>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">netstandard1.3</PackageTargetFramework>
-    <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true'">win</PackageTargetRuntime>
-    <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime>
+    <PackageTargetFramework Condition="'$(TargetsUnix)' == 'true'">netstandard1.3</PackageTargetFramework>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <PackageDestination Include="runtimes/win/lib/netstandard1.3">
+      <TargetFramework>netstandard1.3</TargetFramework>
+    </PackageDestination>
+    <!-- need to support rid-less restore for packages.config -->
+    <PackageDestination Include="lib/net46">
```

Comment:
It doesn't explicitly build for desktop and we want the windows NETStandard to work on desktop projects that use packages.config which cannot understand RID-specific assets.